Makefile

```diff
@@ -15,6 +43,10 @@ PMM_RELEASE_VERSION ?=
 PMM_RELEASE_TIMESTAMP ?= $(shell date '+%s')
 PMM_RELEASE_FULLCOMMIT ?= $(shell git rev-parse HEAD)
 PMM_RELEASE_BRANCH ?= $(shell git describe --always --contains --all)
+PATH_TO_KUBECONFIG ?= ${HOME}/.kube/config_eks
```

Comment:
I'd prefer to have as a default -  the default path to kubeconfig `${HOME}/.kube/config`